src/components/ProfilePage/ProfilePage.js

```diff
@@ -1,26 +1,31 @@
 import './profilePage.css';
 import default_user from '../../assets/images/default-user.jpg';
 import Avatar from './Avatar/Avatar';
-import React, { useRef, useState} from 'react';
+import React, { useRef, useState } from 'react';
 import $ from 'jquery';
 
 const ProfilePage = (props) => {
     const ref_image = React.createRef();
     const ref_email = useRef(null);
     const ref_firstName = useRef(null);
     const ref_lastName = useRef(null);
-    const ref_newPassword = useRef(null);
     const user = JSON.parse(sessionStorage.getItem('user'));
-    
+
+    const fieldHandling = (field, value) => {
+        if (field != null)
+            return (value.length === 0 || user[field] === value) ? null : value;
+        else return value.length === 0 ? null : value;
+    }
 
```

Comment:
fieldHandling = (field, value) => {
     retrun value.length === 0 ? null : value 
}

What is the purpose to check the field? 
Why password property is null in update_user?

Reply:
The user can change the password or leave old